doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -412,7 +412,9 @@ Security libraries
 Modem libraries
 ---------------
 
-|no_changes_yet_note|
+* :ref:`modem_key_mgmt` Library:
```

Comment:
Please take this out from this PR and add it as a comment to the 2.9.0 rel note https://github.com/nrfconnect/sdk-nrf/pull/19240.

Reply:
Done.